RocketChat/Rocket.Chat
# chore: Change price filters for Enterprise apps

Co-authored-by: Matheus Lucca do Carmo <mematheuslc@gmail.com>







## Proposed changes (including videos or screenshots)


* Currently, Enterprise apps are classified as “Free apps”. Now, they have been moved to a new “Enterprise” category in the _Apps_ page of the Marketplace.
* Created a new "Enterprise" pricing filter in the _Apps_ page.
* Removed the "Free" tag next to the _install_ button for enterprise apps, on the _App Info_ page.

![New-version](https://user-images.githubusercontent.com/84182706/208493531-11fc0153-2c94-4fab-baac-e359ba89ed72.gif)



## Issue(s)


## Steps to test or reproduce


1. Go to the workspace admin panel
2. Click on the “Apps” menu
3. Click on the filter “All prices”
4. Click on an Enterprise app to see the "App Info" page.

![Old-version](https://user-images.githubusercontent.com/84182706/208493573-7ae93fa6-2124-45b3-815e-07ce66b2568c.gif)

## Further comments

No changes were made for the backend logic of the Enterprise apps.

MKP-170
